test/test-mongocrypt.c

```diff
@@ -724,7 +724,14 @@ _test_setopt_kms_providers (_mongocrypt_tester_t *tester)
       {"{'local': {'key': '" EXAMPLE_LOCAL_MATERIAL "', 'extra': 'invalid' }}",
        "Unexpected field: 'extra'"},
       {"{'local': {'key': 'AAAA'}}", "local key must be 96 bytes"},
-   };
+      /* KMIP test cases. */
+      {"{'kmip': { 'endpoint': '127.0.0.1:5696' }}", NULL},
+      {"{'kmip': { }}", "expected endpoint kmip.endpoint"},
+      /* localhost is a valid endpoint for KMIP.
+       * Unlike Azure, GCP, and AWS, applications run their own KMIP servers. */
+      {"{'kmip': { 'endpoint': 'localhost' }}", NULL},
+      {"{'kmip': { 'endpoint': '127.0.0.1:5696', 'extra': 'invalid' }}",
+       "Unexpected field: 'extra'"}};
```

Comment:
```suggestion
      /* KMIP test cases. */
      {"{'kmip': {'endpoint': '127.0.0.1:5696' }}", NULL},
      {"{'kmip': {}}", "expected endpoint kmip.endpoint"},
      /* localhost is a valid endpoint for KMIP.
       * Unlike Azure, GCP, and AWS, applications run their own KMIP servers. */
      {"{'kmip': {'endpoint': 'localhost' }}", NULL},
      {"{'kmip': {'endpoint': '127.0.0.1:5696', 'extra': 'invalid' }}",
       "Unexpected field: 'extra'"}};
```
An attempt to maintain formatting consistency with other strings.